src/components/TextInput/BaseTextInput.js

```diff
@@ -395,7 +395,7 @@ function BaseTextInput(props) {
                         setTextInputHeight(e.nativeEvent.layout.height);
                     }}
                 >
-                    {props.value || props.placeholder}
+                    {props.value ? `${props.value}\u200B` : props.placeholder}
```

Comment:
Let's add some comment to explain why the change was needed.

@s-alves10 Why did we use `\u200B` instead of `\n`?